fission/package.go

```diff
@@ -387,3 +387,30 @@ func pkgDelete(c *cli.Context) error {
 
 	return nil
 }
+
+func pkgRetry(c *cli.Context) error {
```

Comment:
Might be nice to have a --wait option.  (But no need to block this pr for it, it could be done later)

Reply:
Just opened an issue https://github.com/fission/fission/issues/848 to track this. I will also address https://github.com/fission/fission/issues/816 in same PR as well.